elixir/apps/api/lib/api/gateway/views/client.ex

```diff
@@ -1,6 +1,17 @@
 defmodule API.Gateway.Views.Client do
   alias Domain.Clients
 
+  def render(%Clients.Client{} = client, preshared_key) do
+    %{
+      id: client.id,
+      public_key: client.public_key,
+      preshared_key: preshared_key,
```

Comment:
@AndrewDryga It looks like this is no longer base64 encoded in the new version. Was this on purpose? What is the new encoding?